pydruid/db/sqlalchemy.py

```diff
@@ -123,7 +123,9 @@ def create_connect_args(self, url):
             'path': url.database,
             'scheme': self.scheme,
             'context': self.context,
+            'header': url.query.get('header', 'false') == 'true',
```

Comment:
Nit: no need for the second argument `'false'` here.

Reply:
Correct. I had originally written this thinking the header value would be a boolean.